nextflow_schema.json

```
@@ -1,34 +1,81 @@
+                    "exists": true,
+                    "schema": "assets/schema_input.json",
                     "mimetype": "text/csv",
                     "pattern": "^\\S+\\.csv$",
-                    "schema": "assets/schema_input.json",
                     "description": "Path to comma-separated file containing information about the samples in the experiment.",
-                    "help_text": "You will need to create a design file with information about the samples in your experiment before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row. See [usage docs](https://nf-co.re/magmap/usage#samplesheet-input).",
+                    "help_text": "You will need to create a design file with information about the samples in your experiment before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row.",
                     "fa_icon": "fas fa-file-csv"
                 },
-                "outdir": {
+                "genomeinfo": {
                     "type": "string",
-                    "description": "Path to the output directory where the results will be saved.",
-                    "default": "./results",
-                    "fa_icon": "fas fa-folder-open"
+                    "format": "file-path",
+                    "mimetype": "text/tsv",
+                    "fa_icon": "fas fa-dna",
+                    "description": "Path to comma-separated file containing information about the genomes to map to.",
+                    "help_text": "This has to be a comma-separated file with 3 columns, and a header row. Columns are: accno, genome_fna, genome_gff. Please see the documentation for more information.",
+                    "schema": "assets/schema_genomeinfo.json"
+                },
+                "gtdb_metadata": {
+                    "type": "string",
+                    "description": "Comma-separated list of paths to tab-separated files containing information from GTDB. For column description, see docs.",
+                    "help_text": "Typically, the files are the standard GTDB metadata files, e.g. `bac120_metadata_r226.tsv.gz`. You will need to use or create a file with information about the genomes' taxonomy you want to map to before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 9 columns, and a header row. Columns are: accno, checkm_completeness, checkm_contamination, checkm_strain_heterogeneity, contig_count, genome_size, gtdb_genome_representative, gtdb_representative, gtdb_taxonomy.",
+                    "fa_icon": "fas fa-table"
+                },
+                "checkm_metadata": {
+                    "type": "string",
+                    "description": "Comma-separated list of path to tab-separated files containing the output from CheckM. For column description, see docs.",
```

I dont like the "see docs", because we are here in the "docs" already. Maybe

```suggestion
                    "description": "Comma-separated list of path to tab-separated files containing the output from CheckM. For column description, see [usage docs](https://nf-co.re/magmap/usage).",
```
or even specify the exact headline.
Also true for other occurences.